### DISCUSSION TITLE
Error in Intellij due to "Compiler Output" path not set

### DISCUSSION THREAD

**Discussion by RedstoneLP2** [FabricMC/fabric-api#3143] [Mod Dev Support] [answered]
### Description

Following the "Recommended" Setup steps in the [tutorial](https://fabricmc.net/wiki/tutorial:setup) I encounter the following error after changing "Build and run using" and "Run tests using" to "IntelliJ IDEA" and trying to run "Minecraft Client" through IntelliJ:

"The output path is not specified for module $ModuleName"

A quick internet search showed that it's because of the "Compiler Output" path not being set in IntelliJ.

I haven't yet found any info as to how to fix this for modding with fabric. In the meantime switching the options back to Gradle makes the project compile.
I've also managed to reproduce the issue on a completely clean setup using the ExampleMod.

Any help welcome

**Comment by Benonardo** [comment, marked as answer]
It's in File -> Project Structure at the bottom. You should likely set it to a folder named `out` in your project root.

**Reply by RedstoneLP2** [reply]
Thanks, that worked
